src/main/resources/templates/email/footer.ftlh.mjml

```diff
@@ -1,6 +1,6 @@
 <!-- <#-- @ftlvariable name="" type="com.terraformation.backend.email.model.EmailTemplateModel" --> -->
 <mj-section padding="0px" mj-class="footer-wrapper">
     <mj-column>
-        <mj-text mj-class="text-footer">${strings("notification.email.html.footer.1")}</mj-text>
+        <mj-text mj-class="text-footer"><span>&#169;</span>${strings("notification.email.html.footer.1")}</mj-text>
```

Comment:
This seems fine for now. At some point we should figure out why it wasn't working to have the copyright symbol (rather than the `&#169;`) in the strings file; that ought to have worked.

Reply:
It worked locally, but not in staging